src/components/manage/Sidebar/ObjectBrowserNav.jsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { Button, Segment } from 'semantic-ui-react';
+import { Button, Segment, Popup, Icon as OldIcon } from 'semantic-ui-react';
```

Comment:
Icons from SemanticUI do not exist!!! 😊 Can we change it with the Pastanaga one? (@plone/volto/icons/home.svg)

Reply:
<img width="182" alt="image" src="https://user-images.githubusercontent.com/486927/81586905-8f8dfe80-93b6-11ea-8659-ad8892a695ac.png">

The icon and the text in the popup is a bit off, maybe changing to the SVG helps with alignment.

Reply:
Baseline alignment I guess it will look fine.

Reply:
@sneridagh i don't think it's a breaking change, because this widget didn't not exists before. ObjectBrowser is used in blocks, for example in ImageBlocks, but is not used as a widget now, it's used like an 'addon' of TextWidget. If you think it's better to have a different default value for ObjectBrowser widget, we can change it!

Reply:
Oh ok, it's fine then!